api/client/proto/authservice.proto

```diff
@@ -680,6 +693,30 @@ message GetMFADevicesRequest {}
 // GetMFADeviceResponse is a response for GetMFADevices RPC.
 message GetMFADevicesResponse { repeated types.MFADevice Devices = 1; }
 
+// UserSingleUseCertsRequest is a request for a single-use user certificate.
+message UserSingleUseCertsRequest {
+    oneof Request {
+        UserCertsRequest Init = 1;
+        MFAAuthenticateResponse MFAResponse = 2;
+    }
+}
+
+// UserSingleUseCertsResponse is a response with a single-use user certificate.
+message UserSingleUseCertsResponse {
+    oneof Response {
+        MFAAuthenticateChallenge MFAChallenge = 1;
+        UserCert Cert = 2;
+    }
+}
+
+// UserCert is a single user certificate, either SSH or TLS.
+message UserCert {
+    oneof Cert {
+        bytes SSH = 1;
```

Comment:
Makes sense to add omitempty json tag on these?

Reply:
These are never serialized as JSON